crates/client/data-availability/Cargo.toml

```diff
@@ -43,13 +43,15 @@ blockifier = { workspace = true, default-features = true }
 mc-commitment-state-diff = { workspace = true, default-features = true }
 mc-db = { workspace = true, default-features = true }
 pallet-starknet-runtime-api = { workspace = true, features = ["std"] }
+starknet-core = { workspace = true }
```

Comment:
not needed afaik

Reply:
It can contain some optimization. Or the dependency it has can contain some optimization with "std".
There is no reason not to have at least `features = ["std"]`

Reply:
We should probably change that in all client crates then ?